ota-plus-web/app/views/login.scala.html

```diff
@@ -13,47 +13,50 @@
+                      beginning on @downtimeConfig.startDateTime
+                    </p>
+                </div>
             }
-
             @if(downtimeConfig.isDown) {
-
-            <div class="maintenance-container">
-                <div class="notificaiton">
-                    <div class="logo text-center">
-                        <a href="#"><img src='@routes.Assets.versioned("img/atsgarage_white.png")' id="logo"></a>
-                    </div>
-                    <div class="construction text-center">
-                        <img src='@routes.Assets.versioned("img/icons/construction.png")' id="Construction">
-                    </div>
-                    <div class="header">
-                        <p>ATS Garage is currently offline for scheduled maintenance</p>
+                <div class="maintenance-container">
+                    <div class="notificaiton">
+                        <div class="logo text-center">
+                            <a href="#"><img src='@routes.Assets.versioned("img/atsgarage_white.png")' id="logo"></a>
+                        </div>
+                        <div class="construction text-center">
+                            <img src='@routes.Assets.versioned("img/icons/construction.png")' id="Construction">
+                        </div>
+                        <div class="header">
+                            <p>ATS Garage is currently offline for scheduled maintenance</p>
+                        </div>
                     </div>
                 </div>
-            </div>
-
             } else {
-
-            <div class="container ats">
-                <div class="row">
-                    <div class="login-content">
-                        <div class="ats-header text-center">
-                            <div id="widget-container"></div>
+                <div class="privay-notification">
+                    <div class="content">
+                        <div class="title">
+                            Privacy, data protection and service terms information regarding the sale of ATS:
+                        </div>
+                        <div class="text">
+                            Following the sale of ATS business to HERE Technologies, HERE Technologies assumes responsibility for the provision of the service and personal data of ATS’ customers. HERE Technologies continues to collect and use your personal data in the ways described by HERE Privacy Policy, as available from <a href="https://legal.here.com/privacy" target="_blank">https://legal.here.com/privacy</a> and provides the service in accordance with the General <a href="https://atsgarage.com/en/terms-conditions.html" target="_blank">Terms and Conditions</a>. If you want to contact HERE about your personal data, you can do so by using the address: HERE Global B.V., c/o Privacy, De Run 1115, 5503 LB Veldhoven, Netherlands. We have updated our terms and policies to reflect this change.
```

Comment:
@christianhuggerATS `acquisition of ATS by` rather than `sale of ATS business to` ? can you approve this block?